File: week-1/day-1/substr/tests/test_substr.py
import unittest
from substr import substr


class SubStr(unittest.TestCase):

    def test_substr_happy_case(self):
        self.assertEqual(
            7, substr("this is what I'm searching in", "searching"))

    def test_substr_sad_case(self):
        self.assertEqual(-1, substr("this is what I'm searching in", "not"))

if __name__ == '__main__':
    unittest.main()

File: week-1/day-1/substr/tests/TestSubStr.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestSubStr {

  @Test
  public void testHappyCase() {
    assertEquals(7, SubStr.subStr("this is what I'm searching in", "searching"));
   }

  @Test
  public void testSadCase() {
    assertEquals(-1, SubStr.subStr("this is what I'm searching in", "not"));
   }
}
